library/src/main/res/layout/rxw_list_view_card.xml

```diff
@@ -5,19 +5,20 @@
     android:layout_height="wrap_content"
     android:layout_width="match_parent">
 
-    <LinearLayout
+    <RelativeLayout
         android:layout_height="wrap_content"
-        android:layout_width="match_parent"
-        android:orientation="vertical">
+        android:layout_width="match_parent">
 
-        <LinearLayout
-            android:id="@+id/container"
-            android:layout_height="wrap_content"
+        <android.support.v7.widget.RecyclerView
+            android:id="@+id/recycler_view"
             android:layout_width="match_parent"
-            android:orientation="vertical"/>
+            android:layout_height="wrap_content"
+            android:overScrollMode="never"
+            android:nestedScrollingEnabled="false"/>
```

Comment:
I found a few people having issues when the property was introduced.

The xml attribute was having no affect. I have added a programmatic call to the setNestedScrollingEnabled(false) in ListViewCard just in case.

The only other concern would be that this property is > API 21. I have verified that the property is working on multiple devices below that API level.

We should just make sure that testing is thorough regarding API levels and devices.